cloudprem/main.tf

```diff
@@ -33,21 +33,27 @@ locals {
     Environment = var.environment
   }
 
-  protect_resources = false #var.stack_type == "prod" ? true : false
+  protect_resources = var.stack_type == "production" ? true : false
```

Comment:
What do you think about splitting this out into it's own variable? It would make things a little more explicit i think. We could have terragrunt breakout `protect_resources` in the defaults for an environment? That way we can toggle it intentional from the config side, and it's not implicit on the stack type. I can totally imagine a case where we'd want to QA the "production mode", but still be able to easily destroy that stack when we're done. Thoughts?